### PR TITLE
add conv norm activation test case

### PR DESCRIPTION
**Pull request by LeungChiNan**
## Motivation and Context

add more case for conv,norm,activation function

## Description

add more case for conv,norm,activation function

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: impl/camb/device_configs.py [hunk 1]
```diff
@@ -89,7 +89,7 @@
             args=[
                 {
                     "ins": ["weight"],
-                    "shape": [Skip((2048, 1, 3, 3))],
+                    "shape": [Skip((18, 8, 12, 2)), Skip((6, 9, 3, 5)), Skip((2048, 1, 3, 3)), Skip((2, 6, 2, 3))],
```
Comment (NeosZhang) [inline impl/camb/device_configs.py:92]: 这类skip的形状是无法通过测试吗

Comment (LeungChiNan) [reply impl/camb/device_configs.py:92]: 和其他参数也有关系，不skip会报错
image

**Review by NeosZhang — APPROVED** [review]

File: diopi_test/python/conformance/diopi_configs.py [hunk 2]
```diff
@@ -33,30 +37,78 @@
...
+                    # "shape": ((8, ), (64, ), (96, ), (16, ),
+                    #           (7, ), (15, ), (96, ), (16, )),
                     "shape": ((8, ), (64, ), (96, ), (16, )),
                     "gen_fn": Genfunc.randn,
                 },
             ]
         ),
     ),
 
+    'batch_norm_nan': dict(
```
Comment (yangbofun) [inline diopi_test/python/conformance/diopi_configs.py:72]: batch_norm_nan, nan是指？会产生Nan？

Comment (LeungChiNan) [reply diopi_test/python/conformance/diopi_configs.py:72]: 是的，测试running_var输入负数产生Nan的场景

**Pull request merged**